src/ansys/motorcad/core/geometry.py

```diff
@@ -554,7 +552,11 @@ def rotate(self, centre_point, angle):
         centre_point : Coordinate
             point to rotate Coordinate around.
```

Comment:
```suggestion
            Point to rotate coordinate around.
```

Reply:
Coordinate is a class name so keeping this capitalised

Reply:
```suggestion
            Point to rotate Coordinate around.
```